services/scabbard/src/cli/transaction.rs

```diff
@@ -0,0 +1,402 @@
+// Copyright 2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fmt::Write;
+use std::iter::FromIterator;
+use std::time::Instant;
+
+use protobuf::{self, Message};
+use sabre_sdk::{
+    protocol::{
+        payload::{Action, SabrePayload},
+        ADMINISTRATORS_SETTING_ADDRESS,
+    },
+    protos::IntoBytes,
+};
+use sawtooth_sdk::messages::{
```

Comment:
We can probably use Transact here and the better builder pattern.

Reply:
Created stories to implement all necessary capabilities for Transact and update this to use the Transact builder pattern